samples/feature_flag_with_azure_app_configuraiton_sample.py

```diff
@@ -0,0 +1,43 @@
+from microsoft.featuremanagement import FeatureManager
+from random_filter import RandomFilter
+from azure.appconfiguration.provider import load, SettingSelector
+import os
+from sample_utilities import get_authority, get_audience, get_credential, get_client_modifications
+import json
+import os
+import sys
+from time import sleep
+
+endpoint = os.environ.get("APPCONFIGURATION_ENDPOINT_STRING")
+authority = get_authority(endpoint)
```

Comment:
Why include sample_utilities and all of these get_authority, get_audience, get_credential, get_client_modifications in this sample? I think it's a helpful sample for AAD, but it makes this sample seem heavy. Especially when the util functions have their splitting logic.

Reply:
@rossgrambo, so you're just saying use a connection string?

Reply:
Yeah, I think so. This could be valuable to offer as well- but maybe this one should be "feature_flag_with_**aad**_app_configuration_sample.py" and then offer a simpler one that just uses connection string. 

Or leave out the aad auth and link to the python provider's example for how to use aad.

Reply:
I want a runnable sample, switching to connection string.